include/linux/osd.h

```diff
@@ -54,9 +54,9 @@ static inline int ofi_shm_remap(struct util_shm *shm,
 }
 
 
-#ifdef HAVE_LINUX_PERF_EVENT_H
+#ifdef HAVE_LINUX_PERF_RDPMC
```

Comment:
This needs to be `#if`, not `#ifdef`.  Same everywhere this occurs.

Reply:
Done